content/english/security-fundamentals/what-is-security.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "What is Security?"
+description: ""
+draft: false
+weight: 1
+---
+
+## General Security vs. Cyber Security
+**Security** on its own is defined as the state of being free from danger or threat. This is the same as the definition of **cyber security** except that cyber security is all about creating a state of being free from danger but in **cyberspace**. 
+
+Someone who works in **cybersecurity** is always working on something to make sure that people who use computers are safe when they use the internet in a variety of ways. Some cybers security engineers ensure that users are safe by always being on alert for new vulnerabilities while others do the same job by trying to break programs before hackers can.
+
+## What is cyberspace?
+**Cyberspace** is anywhere that communication happens over a computer network. This can mean when you are interacting with others online in places like social media sites or forums, but it also includes computers talking to each other over the internet or smaller networks for places like a home or office.
+
+## Why do we need security?
+We need security because there are constantly hackers out there trying to steal your information. Cyberspace should be a place where people can visit *without* having to worry about the threat of having their information stolen or their computer put at risk by viruses and malware.
+
+The role of cyber security engineers and the field of cyber security in general is important because there are very real consequences to cybercrimes. On one hand, people’s data, lives, or businesses can be **ruined** by the actions of cyber criminals or hackers. On the other hand, there is also a big financial risk as well. According to the Center for Strategic and International Studies, cybercrime cost the world about **$600 Billion** in 2018. Other sources claim that the number is closer to **$1 Trillion** in 2020 and 2021.
```

Comment:
```suggestion
The role of cyber security engineers and the field of cyber security in general is important because there are very real consequences to cybercrimes. On one hand, people’s data, lives, or businesses can be **ruined** by the actions of cyber criminals or hackers. On the other hand, there is also a big financial risk as well. According to the Center for Strategic and International Studies, cybercrime costs the world about **$600 billion USD** in 2018. Other sources claim that the number is closer to **$1 trillion USD** in 2020 and 2021.
```